samplers/samplers.go

```diff
@@ -308,47 +342,47 @@ func (h *Histo) Flush(interval time.Duration, percentiles []float64, aggregates
 	// we only want to flush the number of samples we received locally, since
 	// any other samples have already been flushed by a local veneur instance
 	// before this was forwarded to us
-	rate := h.localWeight / interval.Seconds()
+	rate := h.LocalWeight / interval.Seconds()
 	metrics := make([]DDMetric, 0, aggregates.Count+len(percentiles))
 
-	if (aggregates.Value&AggregateMax) == AggregateMax && !math.IsInf(h.localMax, 0) {
+	if (aggregates.Value&AggregateMax) == AggregateMax && !math.IsInf(h.LocalMax, 0) {
 		// XXX: why is this recreating tags mutliple times, granted they won't always be used, but it seems preferable to
```

Comment:
as we discussed during pairing, we're copying the tags so that we can avoid aliasing bugs if multiple ddmetric items were sharing the same tag array. (in other words, it's the good old c++ defensive copy) can we replace this comment accordingly?